src/ExportPostman.php

```diff
@@ -74,6 +74,16 @@ public function handle(): void
 
                 if ($structured) {
                     $routeNames = $route->action['as'] ?? null;
+
+                    if (! $routeNames) {
+                        $routeUri = explode('/', $route->uri());
+
+                        // remove "api" from the start
+                        unset($routeUri[0]);
```

Comment:
What if they have a different prefix? We can't lock them down to just having the default one laravel provides.

Reply:
It doesn't matter what the prefix is here, as it will be removed.